dll/win32/shell32/CDefView.cpp

```diff
@@ -389,6 +390,12 @@ CDefView::~CDefView()
 {
     TRACE(" destroying IShellView(%p)\n", this);
 
+    if (m_pShellFolderViewCB)
```

Comment:
Why is this here?
`m_pShellFolderViewCB` is a smart pointer, so no need to manage it by hand?